automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCECustomizationSpec.scala

```diff
@@ -80,14 +80,12 @@ final class NotebookGCECustomizationSpec extends GPAllocFixtureSpec with Paralle
               withNewNotebook(runtime, Python3) { notebookPage =>
                 // Check the extensions were installed
                 val nbExt = notebookPage.executeCell("! jupyter nbextension list")
-                nbExt.get should include("jupyter-gmaps/extension  enabled")
-                nbExt.get should include("pizzabutton/index  enabled")
+                nbExt.get should include("jupyter-iframe-extension/mai enabled")
                 nbExt.get should include("translate_nbextension/main  enabled")
                 // should be installed by default
                 nbExt.get should include("toc2/main  enabled")
 
                 val serverExt = notebookPage.executeCell("! jupyter serverextension list")
-                serverExt.get should include("pizzabutton  enabled")
```

Comment:
I think the test is still ignored right now https://github.com/DataBiosphere/leonardo/blob/dd5d6447e2ba8d1bf6d680cdb71bfa787df6865e/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCECustomizationSpec.scala#L66

Reply:
Good catch. I think this is more important for the other PR I have made. This one is just to have jenkins test this start script issue since it looks fine on local